test/plugin/test_out_detect_exceptions.rb

```diff
@@ -167,7 +167,7 @@ def test_flush_after_max_lines
       feed_lines(d, t, PYTHON_EXC, JAVA_EXC)
     end
     expected =
-      [PYTHON_EXC.lines[0..2].join] + PYTHON_EXC.lines[3..-1] + [JAVA_EXC]
+      [PYTHON_EXC.lines[0..1].join] + PYTHON_EXC.lines[2..-1] + [JAVA_EXC]
```

Comment:
Thanks for adjusting the parameter descriptions... You might still add a comment here saying something like _Check that only the first 2 lines of the trace are kept_. It's also not entirely clear why you didn't also expect to detect the first 2 lines of the Java trace, which might merit another comment.